package.json

```diff
@@ -1,11 +1,35 @@
 {
-  "name": "msal-js-devapp-b2c",
+  "name": "active-directory-b2c-javascript-msal-singlepageapp",
   "version": "1.1.0",
   "license": "MIT",
   "main": "server.js",
   "dependencies": {
     "express": "^4.12.3",
     "morgan": "^1.5.2",
     "path": "^0.11.14"
-  }
+  },
+  "scripts": {
+    "start": "nodemon server.js"
```

Comment:
Given `nodemon` is a dev dependency, I think you should have to scripts, one for dev and one for prod:

```json
{
  "scripts": {
      "start": "node server.js",
      "dev": "nodemon server.js"
   }
}
```